dev-tools/composer.lock

```diff
@@ -0,0 +1,4615 @@
+            ],
+            "authors": [
+                {
+                    "name": "Bernhard Schussek",
+                    "email": "bschussek@gmail.com"
+                }
+            ],
+            "description": "A PHP implementation of Ant's glob.",
+            "support": {
+                "issues": "https://github.com/webmozarts/glob/issues",
+                "source": "https://github.com/webmozarts/glob/tree/4.4.0"
+            },
+            "time": "2021-10-07T16:13:08+00:00"
+        },
+        {
+            "name": "webmozart/path-util",
+            "version": "2.3.0",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/webmozart/path-util.git",
+                "reference": "d939f7edc24c9a1bb9c0dee5cb05d8e859490725"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/webmozart/path-util/zipball/d939f7edc24c9a1bb9c0dee5cb05d8e859490725",
+                "reference": "d939f7edc24c9a1bb9c0dee5cb05d8e859490725",
+                "shasum": ""
+            },
+            "require": {
+                "php": ">=5.3.3",
+                "webmozart/assert": "~1.0"
+            },
+            "require-dev": {
+                "phpunit/phpunit": "^4.6",
+                "sebastian/version": "^1.0.1"
+            },
+            "type": "library",
+            "extra": {
+                "branch-alias": {
+                    "dev-master": "2.3-dev"
+                }
+            },
+            "autoload": {
+                "psr-4": {
+                    "Webmozart\\PathUtil\\": "src/"
+                }
+            },
+            "notification-url": "https://packagist.org/downloads/",
+            "license": [
+                "MIT"
+            ],
+            "authors": [
+                {
+                    "name": "Bernhard Schussek",
+                    "email": "bschussek@gmail.com"
+                }
+            ],
+            "description": "A robust cross-platform utility for normalizing, comparing and modifying file paths.",
+            "support": {
+                "issues": "https://github.com/webmozart/path-util/issues",
+                "source": "https://github.com/webmozart/path-util/tree/2.3.0"
+            },
+            "abandoned": "symfony/filesystem",
+            "time": "2015-12-17T08:42:14+00:00"
+        }
+    ],
+    "aliases": [
+        {
+            "package": "humbug/php-scoper",
+            "version": "9999999-dev",
+            "alias": "0.14.2",
+            "alias_normalized": "0.14.2.0"
+        }
+    ],
+    "minimum-stability": "stable",
+    "stability-flags": {
+        "humbug/php-scoper": 20
+    },
+    "prefer-stable": false,
+    "prefer-lowest": false,
+    "platform": {
+        "php": "^7.4 || ^8.0"
+    },
+    "platform-dev": [],
+    "platform-overrides": {
+        "php": "7.4"
+    },
+    "plugin-api-version": "2.1.0"
```

Comment:
Update your Composer :)

Reply:
@keradus ping ☝🏼

Reply:
@keradus ping ☝🏼

Reply:
i don't see any value in analysing the lock file on this moment and it doesn't bother anyone but you

Reply:
> it doesn't bother anyone but you

I thought you were better than this... In fact I was hoping you would update your Composer and discover by yourself [this](https://getcomposer.org/doc/06-config.md#allow-plugins) new option of Composer 2.2 and update `composer.json` accordingly.

Reply:
you quoted part of my line. let me rephrase - I'm not aware of benefits, you didn't bother to mention any initially, and everyone else already approved the PR.

You want to be picky? tell ppl what to update.
You want to share some interesting idea, give the reasoning from start - it works way better ;)

Reply:
>  tell ppl what to update.

I did :trollface: 

> Update your Composer :)

Well, it will stop working in July, so I guess no need to update it right now :P